src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -0,0 +1,202 @@
+fix *low-hanging fruit* issues; either those the primary is currently managing
+or those left undone from previous incidents. For example, if an incident
+reveals incorrect or missing documentation, the primary should work to fix such
+issues within his/her week. As a rule of thumb, low-hanging fruit is considered
+to be anything that primary believes is fixable within a half-day's (4 hours)
+worth of effort. 
+
+When primary has challenges any of the paths to resolution of an incident, s/he
+should feel free to engage other developers with help. But primary should first
+enlist the backup. When developer expertise other than backup is needed,
+primary should arrange mutually agreeable times to engage those other
+developers.
+
+There shall be no expectation that a developer serving as primary can get any
+other work done beyond their customer response obligations. In slow weeks, its
+conceivable they can. But, there can be no implied assumption or expectation
+that this will be the case. Furthermore, even in a slow week, the primary may
+likely be able to find plenty of *low-hanging-fruit-type* tasks in the main
+repository issues to tackle and fix.
+
+During their communication's with the customer, the primary should avoid
+reply-all-type communications.
+
+For applicable incidents, the primary should prepare a set of bullets to
+form an outline of a *post-mortem* for the incident to be reviewed at VisIt_
+project meetings.
+
+The Backup's Role
+-----------------
+
+Ideally, like the rest of the development team, the backup is never called into
+action for customer response. However, there are cases where the backup may be
+needed. For this reason, the backup is asked to at least track and maintain
+knowledge and awareness of the issues the primary is supporting. A good way to
+achieve this is to selectively *watch* the visit-live-incidents repository.
+In cases where primary needs help in resolving an incident, the backup
+shall serve as the first line of defense. In addition, the backup should be
+ready to step up to the primary role in the (hopefully rare) occurrence that the
+primary is unable to maintain the one-hour response time goal. To the extent
+possible, the transition from backup to primary should be managed formally.
+Otherwise, the backup should not engage in customer response activities.
+
+In the short term, it will help to pair veteran backups with a *primary*
+who may have less experience with live customer response.
+
+Escalation
+----------
+
+Customer response incidents may escalate for a variety of reasons. The 
+technical expertise or authority required may be beyond the primary's abilities.
+Other difficulties may arise in responding to a given customer's needs.
+For issues that the primary does not know how to resolve, the backup should be
+enlisted. It may just be a quick response that the backup can answer. If the
+backup cannot help, the primary should enlist other developers using the @
```

Comment:
In the case where the primary and secondary are facing an issue they don't know how to resolve and must escalate to another developer, how are they to know which developer to tag?

Reply:
tag the whole team. Do we have a team name to tag?

Reply:
I've tried tagging @visit-dav/visit-developers before, but I don't know if it has worked.

Reply:
According to [this ref](https://github.blog/2012-05-09-introducing-team-mentions/), that should work.

Reply:
@ARSanderson can you comment on whether you got subscribed to this issue by @rusu24edward doing a team @ mention?

Reply:
Mark, the only link I have to this pull request is via your @ comment to me. So if I am understanding correctly Eddie did a team @ mention but seems I was not picked to be on the team :-).

Reply:
Ok, thanks @ARSanderson. Out of curiosity, what do you see for the `watching` status if you go to [this page](https://github.com/orgs/visit-dav/teams/visit-developers).

Reply:
Basically a blank page for a discussion. There is following 

> "There are no discussions for this team yet. Start a discussion by typing in the box above."

And I can "Unwatch" but it is not clear what I am watching. I have drug a screen shot in this message of the screen.

![Screen Shot 2020-01-16 at 11 05 57 AM](https://user-images.githubusercontent.com/26089249/72541283-3f5fbf00-3850-11ea-92e1-80bf0d95cb2e.png)